package.json

```diff
@@ -10,10 +10,8 @@
     "cloud-env": "0.2.2",
     "http-proxy": "1.16.2",
     "nodemailer": "4.0.1",
-    "node-static": "0.7.9"
-  },
-  "nonDefaultDependencies": {
```

Comment:
Would it make more sense just to not include it as a dependency in package.json, then warn the user if it's not installed when it's called in `sockets.js` to run `npm install --no-save ofe` if it's desired? I don't really like the idea of using `nonDefaultDependencies` when it's not a valid `package.json` field